Automattic/wp-calypso
# Support Contact: Don't let users submit questions to the Hebrew forums

#### Changes proposed in this Pull Request

* Since our Hebrew forums are not monitored, we'll ask our Hebrew users to post in English to the English forums.

#### Testing instructions

* Set your language to Hebrew and use an account that is inelegible to get Livechat support. You should get to a form like this (the English screen text will be translated):
<img width="753" alt="Screenshot 2020-01-15 at 12 35 39" src="https://user-images.githubusercontent.com/203408/72431141-5e008000-3794-11ea-93ea-c038a1a418d3.png">

Requires D37706-code to actually not submit to the Hebrew forums.
